src/zzz_od/operation/enter_game/open_game.py

```
@@ -49,3 +53,15 @@ def open_game(self) -> OperationRoundResult:
         )
 
         return self.round_success(wait=5)
+
+    def _open_game_via_steam(self) -> OperationRoundResult:
+        app_id = self.ctx.game_account_config.steam_app_id
+        if app_id == '':
+            return self.round_fail('未配置 Steam App ID，请前往 [ 账户管理 ] 设置')
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)